canvasapi/grading_period.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from six import python_2_unicode_compatible
+
+from canvasapi.canvas_object import CanvasObject
+from canvasapi.exceptions import RequiredFieldMissing
+from canvasapi.util import combine_kwargs
+
+
+@python_2_unicode_compatible
+class GradingPeriod(CanvasObject):
+
+    def __str__(self):
+        return '{} ({})'.format(self.title, self.id)
+
+    def update(self, course_id, grading_period, **kwargs):
```

Comment:
Is `course_id` actually required from the user? Shouldn't it already be in the object?